pytrendy/post_processing/segments_refine/gradual_expand_contract.py

```
@@ -88,3 +88,57 @@ def _get_window_df(center: str, days: int = 7) -> pd.DataFrame:
             update_next_segment(i, new_end, segments, segments_refined)
 
     return segments_refined
+
+
+def pad_gradual_trends(df: pd.DataFrame, value_col: str, segments: list[dict], method_params: dict) -> list[dict]:
```

Ruff (0.15.21)

[warning] Unused function argument: `value_col`

(ARG001)